internal/core/items/flickable.rs

```diff
@@ -251,9 +251,7 @@ impl FlickableData {
                     InputEventFilterResult::ForwardEvent
                 }
             }
-            MouseEvent::Wheel { position, .. } => {
-                InputEventFilterResult::InterceptAndDispatch(MouseEvent::Moved { position })
-            }
+            MouseEvent::Wheel { .. } => InputEventFilterResult::ForwardEvent,
```

Comment:
I think this means that the docs should be changed for the flickable "algorithm" as well. Currently it says:

> 2. Any wheel events are interpreted as attempt of the user to scroll.

I think that section can simply be removed? (And the others renumbered)